portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
+/*
+ * Displays the selected tab on the homepage
```

Comment:
Common good practice for comments is to start with a capital case letter and end in period.